tasks/main.yaml

```diff
@@ -0,0 +1,4 @@
+---
+- name: Place holder task for infoblox role.
```

Comment:
Since ```requests``` is a requirement it might be good to leave a message for user here that it should be installed.